### DISCUSSION TITLE
have error when insert data on Milvus

### DISCUSSION THREAD

**Discussion by BlueTeamQQ1** [milvus-io/milvus#24077] [Q&A and General discussion]
Hi, everyone

When I push the data onto the milvus with batch one, this error appears

![c310087d3121406ca6a76ffb01eed01c](https://github.com/milvus-io/milvus/assets/63582498/1a1d2cb7-d2d7-4d20-848e-9e8cb40346ed)

Can you guys help me ? 
Thanks all

**Comment by yhmo** [comment]
The error "force to deny ..." indicates quota limit is triggered for intensive insert operations. Typically, this error occurs when user continually/repeatly calls insert() for long time, the large amount of insert requests exceed the consuming ability of milvus.

**Reply by BlueTeamQQ1** [reply]
how to solve this problem?

**Reply by BlueTeamQQ1** [reply]
within 1 second, I call insert function at least 1 time

**Reply by xiaofan-luan** [reply]
check your memory. The quota limitation usually happens if 
1. you don't have enough memory
2. Datanode didn't catch ingestion rate
